packages/compass-schema/src/utils.ts

```diff
@@ -0,0 +1,16 @@
+import type { AtlasClusterMetadata } from '@mongodb-js/connection-storage/renderer';
+
+export function getAtlasPerformanceAdvisorLink({
+  clusterId,
+  clusterType,
+  clusterName,
+}: Pick<AtlasClusterMetadata, 'clusterId' | 'clusterType' | 'clusterName'>) {
```

Comment:
This looks correct to me, cc @yx2387  to confirm my understanding of PA is correct and that this isn't about to change